src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/TranslatedDocument.java

```diff
@@ -24,6 +25,7 @@ public class TranslatedDocument {
     public DocumentType getCorrespondingDocumentType() {
         switch (documentType) {
             case DEFENDANT_RESPONSE : return DEFENCE_TRANSLATED_DOCUMENT;
+            case CLAIM_INTENTION : return CLAIM_INTENTION_TRANSLATED_DOCUMENT;
```

Comment:
please check if it should be "CLAIMANT_INTENTION" or "CLAIM_INTENTION" here and all other places

Reply:
Good spot Manish.  Confirmed with Kiera it should be CLAIMANT_INTENTION.  The ticket was incorrect.